internal/common/enums.rs

```diff
@@ -356,6 +356,8 @@ macro_rules! for_each_enums {
                 /// The role for widget with editable text such as a
                 /// [`LineEdit`](../widgets/lineedit.md) or a [`TextEdit`](../widgets/textedit.md)
                 TextInput,
+                /// The element is a [`Switch`](../widgets/switch.md) or behaves like one.
```

Comment:
I added this comment for consistency, but the link is not pointing anywhere AFAIK. Not sure how to fix it or if it is even meant to be clicked through docs.rs.

Reply:
The links do work in this page, so that's correct.